app/lib/screens/history/history.dart

```diff
@@ -23,6 +60,22 @@ class _TravelHistoryState extends State<TravelHistory> {
           style: Theme.of(context).textTheme.headline5,
         ),
       ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () async {
+          buildPdf();
+          savePdf();
+          final output = await getTemporaryDirectory();
+          String fullPath = ("${output.path}/example.pdf");
+
+          Navigator.push(
+              context,
+              MaterialPageRoute(
+                  builder: (context) => PdfPreview(
+                        path: fullPath,
+                      )));
+        },
+        child: Icon(Icons.save),
+      ),
```

Comment:
Is travel history the only screen where we can generate pdfs? If so, we can extract this widget and put it in `lib/screens/local_wdget`

Reply:
Probably not only in `TravelHistory`. There could be a case where a partial form could be emailed. Maybe this can go under `common/utility`

Reply:
No. But I create different class to generate the pdf file. I did this just for testing purposes. This class will go in different folder and we can use that class whenever we needed. I will fix it.